Lib/test/test_memoryio.py

```diff
@@ -463,6 +465,25 @@ def test_getbuffer(self):
         memio.close()
         self.assertRaises(ValueError, memio.getbuffer)
 
+    def test_getbuffer_gc_collect(self):
+        memio = self.ioclass(b"1234567890")
+        buf = memio.getbuffer()
+        memiowr = weakref.ref(memio)
+        bufwr = weakref.ref(buf)
+        # Create a reference loop.
+        a = [buf]
+        a.append(a)
+        # XXX The Python implementation emits an unraisable exception.
```

Comment:
It is a different issue that exists also in 3.11.

Reply:
`XXX` is not very useful. What is it supposed to mean? "TODO" or "Note"? Please write out the intended message using words.

Reply:
See #111330.

Reply:
I know that there is a lot of XXX comments. They are all equally confusing; is it a TODO comment? Is it a warning? Is it a future enhancement? IMO, we don't need more such confusing letters. If there are no specific meaning you wish to communicate, I suggest to remove the XXX characters.

Reply:
I am confusing. What exactly do you want to add to this comment?

I do not want to remove the comment, because it is useful. It explains non-obvious code. I do not want to remove only XXX, because the current behavior is not normal. It should be fixed, and I do not want the comment and no longer needed context manager were kept after fixing the bug. XXX is a reminder to me or other coredevs.

Reply:
```suggestion
        # NB: The Python implementation emits an unraisable exception.
```

```suggestion
        # Note: The Python implementation emits an unraisable exception.
```

```suggestion
        # The following code is not obvious because the current behaviour is not normal;
        # The Python implementation emits an unraisable exception.
```